cisco_sdwan/manifest.json

```
@@ -0,0 +1,55 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "2da35edb-5e33-4e5f-93c3-65e08478d566",
+  "app_id": "cisco-sdwan",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor your Cisco SD-WAN environment with Datadog.",
+    "title": "Cisco SD-WAN",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Network"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "auto_install": true,
+      "source_type_id": 12300976,
+      "source_type_name": "Cisco SDWAN",
+      "configuration": {},
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "cisco_sdwan.",
+        "check": "cisco_sdwan.device.reachable",
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    },
+    "dashboards": {
+      "Cisco SD-WAN": "assets/dashboards/cisco_sd-wan.json"
+    },
+    "monitors": {
+      "[Cisco SD-WAN] Device Unreachable Alert": "assets/monitors/device_unreachable.json",
+      "[Cisco SD-WAN] Tunnel Down Alert": "assets/monitors/tunnel_down.json",
+      "[Cisco SD-WAN] Device Reboot Alert": "assets/monitors/device_reboot.json"
+    }
+  },
+  "author": {
+    "support_email": "help@datadoghq.com",
+    "name": "Datadog",
+    "homepage": "https://www.datadoghq.com",
+    "sales_email": "info@datadoghq.com"
+  },
+  "oauth": {}
```

```suggestion
  }
```
This field isn't needed anymore. Non blocking